guides/common/modules/proc_updating-disconnected-server.adoc

```diff
@@ -74,7 +80,7 @@ sslverify = 1
 +
 [options="nowrap" subs="attributes"]
 ----
-# reposync --delete --download-metadata -p ~/{project-context}-repos -n \
+# dnf reposync --delete --download-metadata -p ~/{project-context}-repos -n \
```

Comment:
```suggestion
# dnf reposync \
--delete \
--download-metadata \
-p ~/{project-context}-repos \
-n \
```

We generally have each argument on its own line. I am unsure if my proposed change makes things better or just more consistent. :shrug: